src/box/applier.cc

```diff
@@ -369,9 +369,23 @@ applier_connection_init(struct iostream *io, const struct uri *uri,
 	}
 }
 
+/**
+ * Determine if any of the remote peers is already booted, in which case this
```

Comment:
I wouldn't say "any of the remote peers". You check here just one peer - this single applier.

Reply:
Fixed:
```diff
diff --git a/src/box/applier.cc b/src/box/applier.cc
index 5eaa18e24..42819f6a6 100644
--- a/src/box/applier.cc
+++ b/src/box/applier.cc
@@ -370,8 +370,8 @@ applier_connection_init(struct iostream *io, const struct uri *uri,
 }
 
 /**
- * Determine if any of the remote peers is already booted, in which case this
- * is not a bootstrap but rather a join to an existing replica set.
+ * Determine if the remote peer is already booted, in which case this is not a
+ * bootstrap but rather a join to an existing replica set.
  */
 static void
 applier_check_join(const struct applier *applier)

```